front/src/Components/ReportMenu/BlockSubMenu.svelte

```diff
@@ -0,0 +1,43 @@
+<script lang="ts">
+    import {blackListManager} from "../../WebRtc/BlackListManager";
+    import {showReportScreenStore, userReportEmpty} from "../../Stores/ShowReportScreenStore";
+    import {onMount} from "svelte";
+
+    export let userUUID: string | undefined;
+    let userIsBlocked = false;
+
+    onMount(() => {
+        if (userUUID === undefined) {
+            userIsBlocked = false;
+            console.error("There is no user to block");
+        } else {
+            userIsBlocked = blackListManager.isBlackListed(userUUID);
+        }
+    })
+
+    function blockUser(): void {
+        if (userUUID === undefined) {
+            console.error("There is no user to block");
+            return;
+        }
+        blackListManager.isBlackListed(userUUID)
+            ? blackListManager.cancelBlackList(userUUID)
+            : blackListManager.blackList(userUUID);
+        showReportScreenStore.set(userReportEmpty); //close the report menu
+    }
+</script>
+
+<div class="block-container">
+    <h3>Block</h3>
+    <p>Block any communication from and to this user. This can be reverted.</p>
```

Comment:
It would be cool to be able to display the name of the user in the message. Like:

>     <p>Block any communication from and to {userName}. This can be reverted